cpp/unreal_plugins/WheeledVehicle/Source/WheeledVehicle/VehicleWheel.cpp

```diff
@@ -0,0 +1,43 @@
+//
+// Copyright(c) 2022 Intel. Licensed under the MIT License <http://opensource.org/licenses/MIT>.
+//
+
+#include "WheeledVehicle/VehicleWheel.h"
+
+#include "CoreUtils/Config.h"
+#include "CoreUtils/Log.h"
+
+UVehicleWheel::UVehicleWheel()
+{
+    SP_LOG_CURRENT_FUNCTION();
+
+    if (!Config::s_initialized_) {
+        return;
+    }
+    
+    // We need to set the torque combine method to override mode for external torque 
+    // values to be effective in applying drive torques and brake torques to the vehicle.
+    ExternalTorqueCombineMethod = ETorqueCombineMethod::Override;
+
+    bAffectedByHandbrake = Config::get<bool>("WHEELED_VEHICLE.VEHICLE_WHEEL.AFFECTED_BY_HANDBRAKE");
```

Comment:
Are all of these needed? Did the Chaos tutorials instruct you to set each of these parameters?

Reply:
I refined these parameters. Chaos tutorials talks about some of these paramters, but I've mainly included the ones that ended up being needed. Also, I think it's good to expose these parameters to the python interface.

Reply:
Ok that sounds good. If you needed to change a parameter to achieve desirable behavior in the sportscar example, or the parameter was mentioned in the Chaos tutorials that we have been following, then I'm in favor of including it here and exposing it via our config system. If neither of these conditions are true for a particular parameter, then I would be in favor of removing that parameter. Our goal should be to expose the minimal set of parameters we actually need, not to expose the gazillions of Unreal parameters that we don't need to think about.